admin/class-import-wpseo.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * @package WPSEO\Admin\Import\External
+ */
+
+/**
+ * Class WPSEO_Import_WPSEO
+ *
+ * Class with functionality to import Yoast SEO settings from wpSEO
+ */
+class WPSEO_Import_WPSEO extends WPSEO_Import_External {
+
+	/**
+	 * @var array
+	 */
+	private $robot_values = array(
+		1 => array( 'index' => 2, 'follow' => 0 ),
+		2 => array( 'index' => 2, 'follow' => 1 ),
+		3 => array( 'index' => 1, 'follow' => 0 ),
+		4 => array( 'index' => 1, 'follow' => 0 ),
+		5 => array( 'index' => 1, 'follow' => 1 ),
+		6 => array( 'index' => 2, 'follow' => 0 ),
+	);
+
+
+	/**
+	 * Import All In One SEO settings
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->import_post_metas();
+		$this->import_taxonomy_metas();
+
+		$this->set_msg( __( 'wpSEO data successfully imported. ', 'wordpress-seo' ) );
+	}
+
+	/**
+	 * Import the post meta values
+	 */
+	private function import_post_metas() {
+		WPSEO_Meta::replace_meta( '_wpseo_edit_title', WPSEO_Meta::$meta_prefix . 'title', $this->replace );
```

Comment:
I'd prefer that, when we call functions that belong to another class and we call multiple times, we explain what kind of parameters you are sending along. This way we know in one glance what is going on in this function. It increases readability. 
@omarreiss, could you please give your opinion on this?